docs/modules/operation/pages/thresholds/basic.adoc

```diff
@@ -0,0 +1,19 @@
+
+[[threshold-bw]]
+= Basic Walk-Through – Thresholding
+
+This section describes how to create a basic threshold for a single, system-wide variable: the number of logged-in users.
+Our threshold will tell {page-component-title} to create an event when the number of logged-in users on the device exceeds two, and re-arm when it falls below two.
+
+Before creating a threshold, you need to make sure you are collecting the metric against which you want to threshold.
+
+[[metric-collect]]
+== Determine you are collecting metric
+In this case, we have chosen a metric (number of logged-in users) that is collected by default.
+We are also using data collected via SNMP. (For information on other collectors, see <<performance-data-collection/collectors/collectors.adoc#collectors-overview,Collectors>>.)
```

Comment:
Should link to Reference/Collectors if merged after https://github.com/OpenNMS/opennms/pull/3630

Reply:
This will be merged before 3630. Will have to update a lot of those links after 3630 merges. Hoping the link validator catches them.